### PR TITLE
Use 'ent' as Enterprise Search short name

### PR DESCRIPTION
**Pull request by sebgl**
In the codebase we used either 'ents', 'entsearch' or 'ent'. 'ent' is
the short name used in the product api path (`/ent/`), let's stick with
it everywhere.

### CODE REVIEW

**Comment by sebgl** [comment]
Jenkins test this please.

File: config/webhook/manifests.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ webhooks:
       namespace: system
       path: /validate-enterprisesearch-k8s-elastic-co-v1beta1-enterprisesearch
   failurePolicy: Ignore
-  name: elastic-entsearch-validation-v1beta1.k8s.elastic.co
+  name: elastic-ents-validation-v1beta1.k8s.elastic.co
```
Comment (thbkrkr) [inline config/webhook/manifests.yaml:88]:
```suggestion
  name: elastic-ent-validation-v1beta1.k8s.elastic.co
```

File: config/samples/enterprisesearch/ent_es.yml [hunk 2]
```diff
@@ -16,9 +16,9 @@ spec:
 apiVersion: enterprisesearch.k8s.elastic.co/v1beta1
 kind: EnterpriseSearch
 metadata:
-  name: entsearch-sample
+  name: ent-sample
 spec:
-  version: 7.6.0
+  version: 7.7.0
```
Comment (thbkrkr) [inline config/samples/enterprisesearch/ent_es.yml:21]: I would keep this upgrade out of this PR or bump the version everywhere.

Comment (sebgl) [reply config/samples/enterprisesearch/ent_es.yml:21]: you're right let's do this in a different PR.

**Pull request merged**